Knoema.Client/Meta/RawDataFilesModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Knoema.Meta
+{
+	public class RawDataFilesModel
+	{
+		public string Dataset { get; set; }
```

Comment:
You send a request with datasetId and then return the same here? Maybe it's unnecessary?

Reply:
With the dataset, the model has a consistent state and also the API was designed at Knoema